UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.Common/HealthCheck/EventHubLoggingHealthClient.cs

```diff
@@ -0,0 +1,46 @@
+﻿
+using Azure.Messaging.EventHubs;
+using Azure.Messaging.EventHubs.Producer;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Options;
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using UKHO.ExternalNotificationService.Common.Configuration;
+using UKHO.ExternalNotificationService.Common.Logging;
+
+namespace UKHO.ExternalNotificationService.Common.HealthCheck
+{
+    [ExcludeFromCodeCoverage]
+    public class EventHubLoggingHealthClient : IEventHubLoggingHealthClient
+    {
+        private readonly IOptions<EventHubLoggingConfiguration> _eventHubLoggingConfiguration;
+
+        public EventHubLoggingHealthClient(IOptions<EventHubLoggingConfiguration> eventHubLoggingConfiguration)
+        {
+            this._eventHubLoggingConfiguration = eventHubLoggingConfiguration;
```

Comment:
`this.` is not needed, please remove:
```suggestion
            _eventHubLoggingConfiguration = eventHubLoggingConfiguration;
```

Reply:
Yes. Changes Done as requested